src/WebJobs.Extensions.OpenAI/Assistants/AssistantService.cs

```diff
@@ -27,7 +30,7 @@ record InternalChatState(string Id, AssistantStateEntity Metadata, List<ChatMess
     /// This number must be small enough to ensure we never exceed a batch size of 100.
     /// </summary>
     const int FunctionCallBatchLimit = 50;
-
+    const string chatStorage = "ChatStorage";
```

Comment:
As a naming convention, use PascalCase for constant names, both fields and local constants.

https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/identifier-names